Unity_sdk_sample/Unity_sdk_sample/Assets/Viveport_SDK_Sample/Script/ViveportSDK_Sample_TopApi.cs

```diff
@@ -71,6 +70,31 @@ private void InitStatusHandler(int code)          // The callback of Api.init()
         }
     }
 
+    private void QueryRunTimeHandler(int code, int nMode)
+    {
+        if (code == SUCCESS)
+        {
+            Viveport.Core.Logger.Log("QueryRunTimeHandler is successful" + code + "Running mode is " + nMode);
+            // nMode = 1 (Viveport Desktop mode), nMode = 2 (Viveport Arcade mode)
+            if (nMode == 1)
+            {
+                // Use Viveport API
+                Debug.Log("VIVEPORT init pass");
+                onInitComplete.Invoke(code, " <color=#009900>ViveportSDK Init is Complete !!</color>");
+            }
+            else
+            {
+                // Use Viveport Arcade API
+                Debug.Log("VIVEPORT Arcade init pass");
+                onInitComplete.Invoke(2, "<color=#990000>ViveportSDK Arcade Init is Complete !!</color>");
```

Comment:
Current runtime mode is Viveport Arcade. Please refer to Viveport Arcade SDK to know the API usage.

Reply:
OK

Reply:
> Please use Viveport Arcade API , this sample code does not support  Viveport Arcade API
Current runtime mode is Viveport Arcade. This sample code does not support  Viveport Arcade API. Please refer to the following link to know its usage.